draft-ietf-rats-corim.md

```diff
@@ -1780,70 +1787,118 @@ Regardless of the specific integrity protection method used, the Evidence's inte
 
-* For each `measurement-map` entry in the measurements list, the i<sup>th</sup> `measurement-map` is copied to the i<sup>th</sup> `element-map` in the `elements` list of the `rv` `addition` ECT.
+* The ECT `cmtype` of the `ev` entry is set to `reference-values`.
 
-* The issuer of the Endorsement conceptual message is copied to the `ev` `addition` ECT authority field.
+* The Reference Values Triple ({{sec-comid-triple-refval}}) `environment-map` is copied to the `environment-map` for both the `rv` `condition` and `rv` `addition` ECTs.
 
-* If the Endorsement conceptual message has a profile, the profile is copied to the `ev` `addition` ECT profile field.
+* For each `measurement-map` entry in the measurements list, the i<sup>th</sup> `measurement-map` is copied to the i<sup>th</sup> `element-map` in the `element-list` of the `rv` `addition` ECT.
+
+* The issuer of the Endorsement conceptual message is copied to the `ev` `addition` ECT `authority` field.
+
+* If the Endorsement conceptual message has a profile, the profile is copied to the `ev` `addition` ECT `profile` field.
 
 #### Endorsement Triples Transformations {#sec-end-trans}
 
 Endorsement Triple Transformation :
 
-{:ett-enum: counter="bar" style="format Step %d."}
+{:ett-enum: counter="ett" style="format Step %d."}
 
 {: ett-enum}
 * An available `ev` entry ({{sec-ir-end-val}}) is allocated.
-The Endorsement Triple ({{sec-comid-triple-endval}}) `environment-map` is copied to the `environment-map` for both the `ev` `condition` and `ev` `addition` ECTs.
 
-* For each `measurement-map` entry in the measurements list, the i<sup>th</sup> `measurement-map` entry is copied to the i<sup>th</sup> entry in the `addition` ECT `elements` list.
+* The ECT `cmtype` of the `ev` entry's `addition` ECT is set to `endorsements`.
 
-* The issuer of the Endorsement conceptual message is copied to the `ev` `addition` ECT authority field.
+* The Endorsement Triple ({{sec-comid-triple-endval}}) `environment-map` is copied to the `environment-map` for both the `ev` `condition` and `ev` `addition` ECTs.
 
-* If the Endorsement conceptual message has a profile, the profile is copied to the `ev` `addition` ECT profile field.
+* For each `measurement-map` entry in the measurements list, the i<sup>th</sup> `measurement-map` entry is copied to the i<sup>th</sup> entry in the `addition` ECT `element-map` of the `element-list`.
+
+* The issuer of the Endorsement conceptual message is copied to the `ev` `addition` ECT `authority` field.
+
+* If the Endorsement conceptual message has a profile, the profile is copied to the `ev` `addition` ECT `profile` field.
 
 Conditional Endorsement Triple Transformation :
 
-> [^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/285
+{:cett-enum: counter="cett" style="format Step %d."}
+
+{: cett-enum}
+* An available `ev` entry ({{sec-ir-end-val}}) is allocated.
```

Comment:
I know the words aren't new in this PR, but what does it mean for the `ev` entry to be "available"? I don't think we need to talk about allocation behavior. Are we not simply populating the appraisal context with this triple as one to be processed against the ACS?

Reply:
Maybe it's ok to remove it?